Plugin/CppApi/source/CoordinateConversionController.cpp

```diff
@@ -100,8 +100,8 @@ CoordinateConversionController::~CoordinateConversionController()
 /*!
   \brief Converts \a notation and updates the \l results property.
 
-  Before calling this method, set the \l inputMode property to the
-  notation type of \a notation, and call \l setSpatialReference property
+  Before calling this method, set the \l inputFormat property to the
+  desired fromat, and call \l setSpatialReference property
```

Comment:
Typo `format`

Also some of the previous wording is confusing, let's clean it up as well. How about this?

```
..., and set the \l spatialReference property ...
```